.travis.yml

```diff
@@ -19,34 +18,20 @@ os:
 
 addons:
   apt:
-    sources:
-    #- ubuntu-toolchain-r-test
     packages:
       - libarchive-dev
       - libpango1.0-dev
-    #- g++-6
-
-#matrix:
-  #include:
-    #- os: osx
-      #install:
-      #script: brew install tesseract --HEAD
-      #cache:
-        #directories:
-          #- $HOME/Library/Caches/Homebrew
-  #allow_failures:
-    #- script: brew install tesseract --HEAD
+      - libicu-dev
 
 cache:
   directories:
   - leptonica-$LEPT_VER
 
 before_install:
-  - if [[ $TRAVIS_OS_NAME == linux   ]]; then LINUX=true; fi
-  - if [[ $TRAVIS_OS_NAME == osx     ]]; then OSX=true; fi
+  - if [[ $TRAVIS_OS_NAME == osx ]]; then brew install libarchive pango cairo icu4c; fi
+  - if [[ $TRAVIS_OS_NAME == osx ]]; then export PKG_CONFIG_PATH=/usr/local/opt/icu4c/lib/pkgconfig:/usr/local/opt/libarchive/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig; fi
```

Comment:
I am not sure whether setting it here is still valid when `cmake` is running. Maybe better set it there to be safe.

Reply:
Is this ok now?

script:
  - if [[ $TRAVIS_OS_NAME == osx ]]; then export PKG_CONFIG_PATH=$(brew --prefix)/opt/icu4c/lib/pkgconfig:$(brew --prefix)/opt/libarchive/lib/pkgconfig:/$(brew --prefix)/opt/libffi/lib/pkgconfig; fi
  - mkdir build